src/test/ui/issue-35976.stderr

```diff
@@ -4,8 +4,10 @@ error: the `wait` method cannot be invoked on a trait object
 24 |     arg.wait();
    |         ^^^^
    |
```

Comment:
This line needs to be deleted due to recent contributions:

```diff
 error: the `wait` method cannot be invoked on a trait object
   --> $DIR/issue-35976.rs:24:9
    |
 24 |     arg.wait();
    |         ^^^^
-   |
 help: another candidate was found in the following trait, perhaps add a `use` for it:
    |
 11 | use private::Future;
    |
 
 error: aborting due to previous error
```

Reply:
done